packages/payment-proxy-client/src/App.tsx

```diff
@@ -416,22 +421,19 @@ export default function App() {
                           </RadioGroup>
                         </FormControl>
                       </Box>
-                      {useMicroPayments && payDisabled && (
-                        <LinearProgressWithLabel
-                          variant="determinate"
-                          color={"primary"}
-                          value={paymentProgress}
-                        />
-                      )}
-
-                      <Button
+                      <ProgressButton
                         variant="contained"
-                        disabled={payDisabled}
                         onClick={handlePayButton}
-                        sx={{ mt: 1, mr: 1 }}
+                        disabled={downloadProgress > 0}
```

Comment:
It might be nice to disable this button immediately instead of waiting for `downloadProgress >0`, since it can take a moment before we make any progress.

Reply:
https://github.com/statechannels/go-nitro/pull/1778